dna/course/zomes/courses/code/src/section/entry.rs

```diff
@@ -0,0 +1,57 @@
+use hdk::{
+    entry_definition::ValidatingEntryType, 
+    holochain_core_types::{dna::entry_types::Sharing, validation::EntryValidationData},
+    holochain_json_api::{error::JsonError, json::JsonString},
+    holochain_persistence_api::cas::content::Address, 
+};
+use holochain_entry_utils::HolochainEntry; 
+
+#[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
+pub struct Section {
```

Comment:
also we need to have `course_anchor_address` here to implement the implicit link from section to course